src/gawronski-bartosz/Fsd.Bartek.Cs2/Fsd.Bartek.Cs2.Ex2/Domain/Fabric.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Fsd.Bartek.Cs2.Ex2.Domain.Animals;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fsd.Bartek.Cs2.Ex2.Domain.UI;
+
+namespace Fsd.Bartek.Cs2.Ex2.Domain
+{
+    public class Fabric
+    {
+        public Cat[] Cat;
```

Comment:
jakbys miał 30 gatunków zwięrząt to każdy by miał swoją tablice? Pamiętasz co to polimorfizm?